lib/components/viewers/stop-viewer.js

```diff
@@ -32,6 +32,12 @@ const Scrollable = styled.div`
   height: 100%;
   overflow-y: auto;
 `
+// Alert with custom styles
+const StyledAlert = styled(Alert)`
+  margin: 5px 0;
```

Comment:
Nit: I think this could use 10px on the top.

Reply:
Updated with https://github.com/opentripplanner/otp-react-redux/pull/292/commits/98767d35ced6328bc487cdf61e47114d73cfd672.